keyboards/nopunin10did/railroad/keymaps/default/keymap.c

```diff
@@ -0,0 +1,86 @@
+// The Railroad Keyboard
+// Copyright 2020 by W. Alex Ronke, a.k.a. NoPunIn10Did (w.alex.ronke@gmail.com)
+// Licensed under GNU GPL v2, as per the QMK firmware project
+
+#include "railroad.h"
```

Comment:
```suggestion
#include QMK_KEYBOARD_H
```

Prefer this generic include over the specific one.

Reply:
Updated manually.